xLights/preferences/SequenceFileSettingsPanel.cpp

```diff
@@ -152,6 +152,7 @@ SequenceFileSettingsPanel::SequenceFileSettingsPanel(wxWindow* parent,xLightsFra
 	Connect(ID_LISTBOX_MEDIA,wxEVT_COMMAND_LISTBOX_SELECTED,(wxObjectEventFunction)&SequenceFileSettingsPanel::OnMediaDirectoryListSelect);
 	Connect(ID_BUTTON_ADDMEDIA,wxEVT_COMMAND_BUTTON_CLICKED,(wxObjectEventFunction)&SequenceFileSettingsPanel::OnAddMediaButtonClick);
 	Connect(ID_BUTTON_REMOVE_MEDIA,wxEVT_COMMAND_BUTTON_CLICKED,(wxObjectEventFunction)&SequenceFileSettingsPanel::OnRemoveMediaButtonClick);
+	Connect(ID_CHOICE_VIEW_DEFAULT,wxEVT_COMMAND_CHOICE_SELECTED,(wxObjectEventFunction)&SequenceFileSettingsPanel::OnViewDefaultChoiceSelect);
```

Comment:
This is within the //*) markers that codeblocks uses.  Thus, if the codeblocks file wasn't updated to include this connection, it will get wiped out the next time the panel is touched in codeblocks.

Reply:
Where is that file?

Reply:
wxs file needs to be updated to include the connection

Reply:
I corrected the spelling error in that file ... it sounds like you are suggesting there is something else that needs to be changed?

Reply:
![image](https://user-images.githubusercontent.com/2293065/152862913-b3a7a605-3721-46cb-bfcc-e8bd50f14235.png)

Reply:
(with the updated spelling)

Reply:
Is the 20.03 macOS version of codeblocks available for download somewhere?  I followed the instructions to build codeblocks that Dan provided in the Readme but I am encountering a build error:
```
../../../../src/include/logmanager.h:37:39: note: parameter of type 'const wxString &' is declared here
    inline wxString F(const wxString &msg, ...)
                                      ^
sc_wxtypes.cpp:357:17: error: no matching member function for call to 'func'
                func(&wxFileName::Normalize, "Normalize").
                ^~~~
```

I built wxWidgets using the instructions and that appears to be ok.  The error occurs during building of codeblocks (make -j 8).

Reply:
Finally got CB to work.

I made the change but CB deletes the following two lines of code and I am wondering if this is ok?
```
	GridBagSizer1->Fit(this);
	GridBagSizer1->SetSizeHints(this);
```

As far as I am aware, I made no changes to this control element in the panel.  I am unfamiliar with CB so I do not understand why this has happened.

Reply:
Its possible someone manually added those lines to the wrong spot.  There are sections in that file that get replaced every time CodeBlocks regenerates.  For code to not get replaced it has to be added outside those comment marks.  I'd look at history to find when those lines were added and if there is a checkin comment about what was being fixed.

Reply:
Looks like those lines were added when Dan refactored the settings menu.  I'm pretty sure he used CodeBlocks to generate that code so not sure why it would not generate those lines anymore.

Reply:
Yeah, looks like he made those changes a couple years ago.  I'll wait for Dan to indicate what he wants me to do ... maybe move them outside the codeblocks tags?

Reply:
BTW, still having problems compiling CB.  I managed to get around the problem with running bootstrap with:
```
export ACLOCAL_FLAGS="-I `wx-config --prefix`/share/aclocal"
```
This environment variable allows bootstrap to find the autoconf macros.  But I am still having a build problem with make:
```
	clang++ -DHAVE_CONFIG_H -I. -I../../src/include  -I/Users/stevenslupsky/Documents/source/codeblocks-r12709/install_dir/lib/wx/include/osx_cocoa-unicode-3.1 -I/Users/stevenslupsky/Documents/source/codeblocks-r12709/install_dir/include/wx-3.1 -D_FILE_OFFSET_BITS=64 -DwxDEBUG_LEVEL=0 -DWXUSINGDLL -D__WXMAC__ -D__WXOSX__ -D__WXOSX_COCOA__  -I../../src/include/scripting/include -I../../src/sdk/wxscintilla/include -I../../src/include  -DCB_AUTOCONF -stdlib=libc++ -DCB_PRECOMP -DPIC -I../../src/include/tinyxml -DTIXML_USE_STL=YES  -stdlib=libc++ -std=c++14 -Winvalid-pch -fPIC -fexceptions -MT printdlg.o -MD -MP -MF $depbase.Tpo -c -o printdlg.o printdlg.cpp &&\
	mv -f $depbase.Tpo $depbase.Po
notebookstyles.cpp:130:40: error: no member named 'GetBitmapFor' in 'wxBitmap'
        const wxBitmap bmp(page.bitmap.GetBitmapFor(wnd));
                           ~~~~~~~~~~~ ^
notebookstyles.cpp:179:36: error: no member named 'GetBitmapFor' in 'wxBitmap'
            bmp = m_activeCloseBmp.GetBitmapFor(wnd);
                  ~~~~~~~~~~~~~~~~ ^
notebookstyles.cpp:181:38: error: no member named 'GetBitmapFor' in 'wxBitmap'
            bmp = m_disabledCloseBmp.GetBitmapFor(wnd);
                  ~~~~~~~~~~~~~~~~~~ ^
notebookstyles.cpp:316:40: error: no member named 'GetBitmapFor' in 'wxBitmap'
        const wxBitmap bmp(page.bitmap.GetBitmapFor(wnd));
                           ~~~~~~~~~~~ ^
notebookstyles.cpp:364:36: error: no member named 'GetBitmapFor' in 'wxBitmap'
            bmp = m_activeCloseBmp.GetBitmapFor(wnd);
                  ~~~~~~~~~~~~~~~~ ^
notebookstyles.cpp:366:38: error: no member named 'GetBitmapFor' in 'wxBitmap'
            bmp = m_disabledCloseBmp.GetBitmapFor(wnd);
                  ~~~~~~~~~~~~~~~~~~ ^
6 errors generated.
make[3]: *** [notebookstyles.o] Error 1
```


I found a precompiled nightly that is working for me.  Could it be a difference in the version of CB that is causing those lines to be deleted?  The nightly I am using is r12705.

Reply:
Ok, I think this is ready to be merged.  I moved the extra two lines Dan added outside the CB tag.